Gopkg.toml

```diff
@@ -81,3 +81,12 @@ required = [
 [[constraint]]
   branch = "master"
   name = "github.com/wadey/gocovmerge"
+
+[[constraint]]
+  branch = "update.status"
+  name = "github.com/redhat-developer/devconsole-api"
+  source = "https://github.com/corinnekrych/devconsole-api.git"
```

Comment:
@corinnekrych shouldn't you use `github.com/redhat-developer/devconsole-git` here ?

Reply:
yes! this is temporary until this associated PR  get merged:https://github.com/redhat-developer/devconsole-api/pull/16 before merging i'll point to master

Reply:
Take it in https://github.com/redhat-developer/devconsole-api/pull/16